test/e2e/specs/editor/blocks/cover.spec.js

```diff
@@ -0,0 +1,242 @@
+		// Otherwise, the URL for the img src attribute is that of a placeholder.
+		await coverBlock.locator( `img[src^=http]` ).waitFor();
+
+		const backgroundImageURL = await coverBlock
+			.locator( 'img' )
+			.getAttribute( 'src' );
+
+		expect( backgroundImageURL ).toContain( testImageFilename );
+	} );
+
+	test( 'dims background image down by 50% by default', async ( {
+		page,
+		editor,
+	} ) => {
+		await editor.insertBlock( { name: 'core/cover' } );
+
+		const testImage = await getTestImage();
+
+		const coverBlock = page.getByRole( 'document', {
+			name: 'Block: Cover',
+		} );
+
+		await coverBlock
+			.getByTestId( 'form-file-upload-input' )
+			.setInputFiles( testImage );
+
+		const [ backgroundDimColor, backgroundDimOpacity ] =
+			await getBackgroundColorAndOpacity(
+				coverBlock.locator( 'span[aria-hidden="true"]' )
+			);
+		expect( backgroundDimColor ).toBe( 'rgb(0, 0, 0)' );
+		expect( backgroundDimOpacity ).toBe( '0.5' );
+	} );
+
+	test( 'can have the title edited', async ( { page, editor } ) => {
+		const titleText = 'foo';
+
+		await editor.insertBlock( { name: 'core/cover' } );
+
+		// Choose a color swatch to transform the placeholder block into
+		// a functioning block.
+		await page
+			.getByRole( 'button', {
+				name: 'Color: Black',
+			} )
+			.click();
+
+		// Activate the paragraph block inside the Cover block.
+		// The name of the block differs depending on whether text has been entered or not.
+		const coverParagraphLocator = page.getByRole( 'document', {
+			name: /Paragraph block|Empty block; start writing or type forward slash to choose a block/,
+		} );
+		await expect( coverParagraphLocator ).toBeEditable();
+
+		await coverParagraphLocator.fill( titleText );
+
+		await expect( coverParagraphLocator ).toContainText( titleText );
+	} );
+
+	test( 'can be resized using drag & drop', async ( { page, editor } ) => {
+		await editor.insertBlock( { name: 'core/cover' } );
+
+		// Open the document sidebar.
+		await editor.openDocumentSettingsSidebar();
+
+		await page
+			.getByRole( 'button', {
+				name: 'Color: Black',
+			} )
+			.click();
+
+		// Open the block list viewer from the toolbar.
+		await page
+			.getByRole( 'toolbar', { name: 'Document tools' } )
+			.getByRole( 'button', { name: 'Document Overview' } )
+			.click();
+
+		// Select the Cover block from the docuemnt overview.
+		await page.getByRole( 'link', { name: 'Cover' } ).click();
+
+		// Ensure there the default value for the minimum height of cover is undefined.
+		await page.getByRole( 'tab', { name: 'Styles' } ).click();
+		const defaultHeightValue = await page
+			.getByLabel( 'Minimum height of cover' )
+			.inputValue();
+		expect( defaultHeightValue ).toBeFalsy();
+
+		// Establish locators for the Cover block and the resize handler.
+		const coverBlock = page.getByRole( 'document', {
+			name: 'Block: Cover',
+		} );
+		const coverBlockResizeHandle = page.locator(
+			'.components-resizable-box__handle-bottom'
```

Comment:
Despite preferring accessible attributes as selectors, it turns out the draggable handler on the bottom edge of the Cover block doesn't have anything that can be used to select it other than the CSS selector.

Reply:
Yeah, it's unfortunate, and we can't add testid either. Adding a comment above explaining the situation should help though!

Reply:
Comment added in 071c31b4fe4685041cbd67bfe90e7d5f27131a0c.